src/libraries/adaptive-curve/ConstantsLib.sol

```diff
@@ -15,11 +19,11 @@ library ConstantsLib {
     int256 public constant TARGET_UTILIZATION = 0.9 ether;
 
     /// @notice Initial rate at target per second (scaled by WAD).
-    int256 public constant INITIAL_RATE_AT_TARGET = int256(0.04 ether) / 365 days;
+    int256 public constant INITIAL_RATE_AT_TARGET = 4_00 * ONE_BPS / 365 days;
 
-    /// @notice Mininimum rate at target per second (scaled by WAD) (0.1% APR).
-    int256 public constant MIN_RATE_AT_TARGET = int256(0.001 ether) / 365 days;
+    /// @notice Minimum rate at target per second (scaled by WAD) (min APR is MAX_RATE_AT_TARGET / CURVE_STEEPNESS).
```

Comment:
`MAX_RATE_AT_TARGET / CURVE_STEEPNESS` is inexact because `/` is not defined as being the wad-based division rounded down

Reply:
writing `MAX_RATE_AT_TARGET over CURVE_STEEPNESS` would imply that it's meta-language and values must be adjusted

Reply:
also it could be written numerically like it was before (0.1% APR)